1_fetch/src/1_fetch.R

```diff
@@ -0,0 +1,14 @@
+library(dplyr)
+library(readr)
+library(stringr)
+library(sbtools)
+library(whisker)
+
+
+# Get the data from ScienceBase
+fetch_data <- function(){
+  out_dir="personal/training/pipelines/ds-pipelines-targets-1/1_fetch/out/"
```

Comment:
Two things related to how you are defining filepaths in your functions:

1. I think this repository should be set up locally as an Rproject, so you should have it open in RStudio as a project (if not, let's connect and figure that out!). One of the best parts about using RStudio projects is that you do not need to specify the full filepaths. The working directory should automatically be set to your project directory when you open your project SO that means that your working directory should be `ds-pipelines-targets-1` and so you can simplify this `out_dir` to just be `1_fetch/out/`! Feels good to remove code, doesn't it :)
2. It is usually a good idea to pass filepaths _**into**_ functions that you create, so that users can see where the file will end up at the top-most script, rather than buried within the function definitions. This makes functions more reusable, too, because you can just pass in a different filepath as the argument and get a new file. So, I would recommend writing this as `fetch_data <- function(out_file)` and having your example use of this function be `fetch_data('1_fetch/out/model_RMSEs.csv')`.

Reply:
1) aha, I see the Rproj file and now it clicked. At first, I was thinking does each folder need it's own RProj. I'm not used to code being inside folders. All is good here. I'll modify the file paths to be relative. 

2) And damn it! I had all the functions take in the file path and then thought, surely these will remain static since we went through all this trouble with the filing system. lol. But I agree, it's nice to show where things are going with the function arguments. I actually had the file paths as defaults in the functions when I was writing them, so we could choose to not specify when calling a function.

Reply:
You are right! The filepaths will _**probably**_ not change, but it is nice to show those top-level rather than inside functions. I think the utility of this practice will be more obvious once we get to the targets code.